server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -64,16 +71,16 @@ public class KillUnusedSegments implements CoordinatorDuty
   private static final Logger log = new Logger(KillUnusedSegments.class);
 
   private final long period;
```

Comment:
Nit: Since we are touching this code anyway, maybe make these fields (`period` and `durationToRetain`) `Duration` instead of `long`. That would make some of the logs more readable.